verticaltabs.js

```diff
@@ -560,6 +556,11 @@ VerticalTabs.prototype = {
       browserPanel.insertBefore(bottom, document.getElementById('fullscreen-warning').nextSibling);
       top.palette = palette;
       this.window.TabsInTitlebar.updateAppearance(true);
+
+      tab_context_menu.removeEventListener('mouseover', enter);
+      tab_context_menu.removeEventListener('mouseout', exit);
+      toolbar_context_menu.removeEventListener('mouseover', enter);
+      toolbar_context_menu.removeEventListener('mouseout', exit);
```

Comment:
I wonder if we should remove the eventListeners from `leftbox`, too (before we set it to null eight lines above here)…  What do you think?